src/source/release-notes.rst

```diff
@@ -0,0 +1,194 @@
+.. index:: Release Notes
+.. _release_notes:
+
+Release Notes
+=============
+
+This section attempts to give a little more insight on what our various releases contain, as compared to previous versions.
+
+Feb 2023: 'Campaigns & Micros'
+------------------------------
+
+Synopsis
+""""""""
+
+* For bid, budgets & spent values we now report in micros: 1 cent == 10000 micros. These fields will now also be named accordingly, mentioning whether they represent a value in cents or micros. This is also represented in the categories data, where the interval of allowed values is defined. Mind, this does not include values like 'AUTOMATIC' or 'UNLIMITED'.
+* Introduced campaigns, campaign ID not (yet) mandatory while placing ads. Campaigns have their own daily/total budgets. Since currently only 1 campaign is allowed per user, this effectively represents the seller's complete budget.
+* Moved price and related fields into a struct.
+* removed externalID, use vendorID instead.
```

Comment:
Maybe bundle the below into "removed fields" bullet list?

Reply:
bullet lists broken in this version of sphinx/docutils 😭 I can make a separate heading, sure.